var/spack/repos/builtin/packages/h5z-zfp/package.py

```diff
@@ -6,15 +6,15 @@
 from spack import *
 
 
-class H5zZfp(MakefilePackage):
+class H5zZfp(CMakePackage):
     """A highly flexible floating point and integer compression plugin for the
        HDF5 library using ZFP compression."""
 
     homepage = "https://h5z-zfp.readthedocs.io/en/latest"
     git      = "https://github.com/LLNL/H5Z-ZFP.git"
     url      = "https://github.com/LLNL/H5Z-ZFP/archive/refs/tags/v1.0.1.tar.gz"
 
-    version('develop', branch='master')
+    version('develop', branch='master', preferred=True)
     version('1.0.1', sha256='b9ed91dab8e2ef82dc6706b4242c807fb352875e3b21c217dd00782dd1a22b24')
```

Comment:
The CMake build wasn't added until after the 1.0.1 release so none of the specified versions are even valid for this package.  @brtnfld can we get a new tag or release of the plugin to use for the package?